Content.Server/Speech/Components/BipolarDisorderAccentComponent.cs

```diff
@@ -0,0 +1,31 @@
+namespace Content.Server.Speech.Components;
+
+/// <summary>
+/// Bipolar disorder accent adds random punctuation ("!", "...", "?!") to the end of phrases, and randomly triggers the emotions selected in the component.
+/// </summary>
+[RegisterComponent]
+public sealed partial class BipolarDisorderAccentComponent : Component
+{
+    /// <summary>
+    /// The probability of triggering an emotion from the "Emotions" list after each phrase.
+    /// </summary>
+    [DataField]
+    public float TriggerEmotionChance = 0.5f;
+    
+    /// <summary>
+    /// The probability of a message being supplemented with punctuation marks.
+    /// </summary>
+    [DataField]
+    public float ChangeMessageChance = 1.0f;
+
+    /// <summary>
+    /// A list of available emotions for playback.
+    /// </summary>
+    [DataField]
+    public List<string> Emotions = new();
```

Comment:
You should change the type so that it validates that the emotions (should probably be renamed to just Emotes for consistancy with other components) are valid IDs.

https://github.com/space-wizards/space-station-14/blob/67d444feb0b47b63a54221a7124e2277a214972f/Content.Server/Chat/EmoteOnDamageComponent.cs#L20-L25